hybrid/hello_ddc.c

```diff
@@ -0,0 +1,25 @@
+/***
+ * This program creates a capability and writes
+ * it to the Default Data Capability register.
+ * Then, it reads and shows the written value.
+ ***/
+
+#include "../include/common.h"
+#include "include/utils.h"
+#include <stdint.h>
+#include <stdlib.h>
+#if !defined(__aarch64__)
+#error "This example is Morello only"
+#endif
+
+int main()
+{
+	pp_cap(read_ddc());
+	uint16_t *some_int_ptr = (uint16_t *) malloc(sizeof(uint16_t));
+	*some_int_ptr = 200;
+	write_ddc((void *__capability) some_int_ptr);
+	uint16_t *__capability ddc_val = (uint16_t *__capability) cheri_ddc_get();
+	pp_cap(cheri_ddc_get());
+	printf("Value in DDC: %d\n", *ddc_val);
```

Comment:
It would be good to make this an `assert`.

Reply:
Yes, as you have suggested we will use assertions instead of prints in every file.